h/api_client/subscribers.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import pyramid.events
+
+import h.api_client.api_client as api_client
```

Comment:
Similarly, I'd suggest that we expose the "public API" of the `h.api_client` package at the top level, so add an `__all__` to __init__.py so that you can do `from h.api_client import Client`.